src/electron/webpack_electron_main.mjs

```diff
@@ -20,33 +20,55 @@ import {fileURLToPath} from 'url';
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = path.dirname(__filename);
 
-export default ({networkStack}) => ({
-  entry: './src/electron/index.ts',
-  target: 'electron-main',
-  node: {
-    __dirname: false,
-    __filename: false,
-  },
-  devtool: 'inline-source-map',
-  module: {
-    rules: [
-      {
-        test: /\.tsx?$/,
-        use: 'ts-loader',
-        exclude: /node_modules/,
-      },
+export default ({networkStack, buildEnv}) => [
```

Comment:
Could you pass the sentry dsn and the app version explicitly so it's clear what's being used?

Reply:
Do you mean directly calling [`getBuildParameters`](https://github.com/Jigsaw-Code/outline-client/blob/master/src/build/get_build_parameters.mjs#L27) and [`getBuildEnvironment`](https://github.com/Jigsaw-Code/outline-client/blob/master/src/build/get_build_environment.mjs#L26) in this file?

Reply:
I see, you mean pass two explicit arguments to this function instead of the big buildEnv object.

Reply:
Yes, pass two explicit arguments. Otherwise you force me to read the code to figure out what from buildEnv you are using.